horace_core/sqw/@sqw/symmetrise_sqw.m

```diff
@@ -216,9 +219,10 @@
 
     elseif iscell(sym)
         % If it's come from SymopRotation.fold or manual equivalent
-        if all(cellfun(@(x) isa(x, {'SymopRotation', 'SymopIdentity'}), sym))
-            if ~sym{1} == SymopIdentity() || ...
-                        isa(sym{1}, 'SymopRotation') && sym{1}.theta_deg ~= 0
+        if all(cellfun(@(x) isa(x, 'SymopRotation') || ...
+                            isa(x, 'SymopIdentity'), sym))
+            if sym{1} ~= SymopIdentity() || ...
+                         isa(sym{1}, 'SymopRotation') && sym{1}.theta_deg ~= 0
```

Comment:
I'm generally not following this. Options for sym{1} are identity or non-zero rotation. Assuming identity as a special case, the remaining sym{2:end} should follow the recipe below on lines 244-7. If sym{1} is a rotation, then for those lines, shouldn't the loop go from 2:fold, not 3:fold? Either way, more explanation in comments please.